RandBLAS/dense.hh

```diff
@@ -177,126 +173,103 @@ DenseSkOp<T>::~DenseSkOp() {
 
 
 
-
-
-
-
-template <typename T, typename T_gen>
-static RNGState gen_unif(
+/** Fill a n by m matrix with random values. If RandBLAS is compiled with
```

Comment:
Let's not say ``n \times m``; just say ``n_rows \times n_cols``.

Reply:
ok

Reply:
resolved in 0220c604b